cmd/crc/cmd/status.go

```diff
@@ -44,18 +44,18 @@ type status struct {
 	Preset           preset.Preset                `json:"preset"`
 }
 
-func runStatus(writer io.Writer, client machine.Client, cacheDir, outputFormat string) error {
+func runStatus(writer io.Writer, client *daemonclient.Client, cacheDir, outputFormat string) error {
 	status := getStatus(client, cacheDir)
 	return render(status, writer, outputFormat)
 }
 
-func getStatus(client machine.Client, cacheDir string) *status {
-	if err := checkIfMachineMissing(client); err != nil {
-		return &status{Success: false, Error: crcErrors.ToSerializableError(err)}
-	}
+func getStatus(client *daemonclient.Client, cacheDir string) *status {
 
-	clusterStatus, err := client.Status()
+	clusterStatus, err := client.APIClient.Status()
 	if err != nil {
+		if strings.HasSuffix(err.Error(), "connect: connection refused") || strings.HasSuffix(err.Error(), "connect: no such file or directory") {
+			return &status{Success: false, Error: crcErrors.ToSerializableError(crcErrors.DaemonNotRunning)}
+		}
```

Comment:
You should replace this with something like because the error you will get error as `*url.Error` in case of daemon service is not running or the sock file doesn't exist.
```golang
var urlError *url.Error
if errors.As(err, &urlError) {
	return &status{Success: false, Error: crcErrors.ToSerializableError(crcErrors.DaemonNotRunning)}
}
```

Reply:
Done, thx!

Reply:
Fwiw, there's a `checkDaemonStarted` in `cmd/crc/cmd/start.go`, we may want to use it more extensively in the future instead of assuming that all `url.Error` mean that the daemon is not running. Just something to have in mind, does not require changes in this PR.